### PR TITLE
SSR-2825_create attachment user guide

### PR DESCRIPTION
**Pull request by h000780**
## Checklist

- [ ] Read CONTRIBUTING.md([for v1](https://github.com/kintone-labs/kintone-ui-component/master/CONTRIBUTING.md), [for v0](https://github.com/kintone-labs/kintone-ui-component/v0_dev/CONTRIBUTING.md))
- [ ] Updated documentation if it is required.
- [ ] Added tests if it is required.
- [ ] Passed lint and test command.

### CODE REVIEW

File: docs/document/docs/components/desktop/attachment.md [hunk 1]
```diff
@@ -1,10 +1,77 @@
...
+| :--- | :--- | :--- | :--- | :--- |
+| options  | object | {} | Object that includes component properties |  |
+
+---
+## Sample Code
+
+> Please check the [package installation](../../getting-started/quick-start.md#installation) method first.
+
+Here is a sample code when all parameters are specified:
+``` javascript
```
Comment (dong0) [inline docs/document/docs/components/desktop/attachment.md:59]: Please refer to the document of other components to add "const Kuc = Kucs['1.x.x'];" here.

File: docs/document/docs/components/desktop/attachment.md [hunk 2]
```diff
@@ -1,10 +1,77 @@
...
+const space = kintone.app.record.getSpaceElement('space');
+const attachment = new Kuc.Attachment({
+  label: 'Attachment',
+  files: [{name: "file.txt", size: "150"}],
```
Comment (dong0) [inline docs/document/docs/components/desktop/attachment.md:63]: Please use ' for uniformity.

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
Please update following the review comments.

File: docs/document/website/versioned_docs/version-1.8.0/components/desktop/attachment.md [hunk 3]
```diff
@@ -7,4 +7,77 @@ original_id: attachment
...
+
+---
+## Sample Code
+
+> Please check the [package installation](../../getting-started/quick-start.md#installation) method first.
+
+Here is a sample code when all parameters are specified:
+``` javascript
+const Kuc = Kucs['1.x.x'];
+const space = kintone.app.record.getSpaceElement('space');
```
Comment (TomokoMiyake) [inline docs/document/website/versioned_docs/version-1.8.0/components/desktop/attachment.md:66]:
```suggestion
const Kuc = Kucs['1.x.x'];

const space = kintone.app.record.getSpaceElement('space');

```

File: docs/document/website/versioned_docs/version-1.8.0/components/desktop/attachment.md [hunk 4]
```diff
@@ -7,4 +7,77 @@ original_id: attachment
...
+const attachment = new Kuc.Attachment({
+  label: 'Attachment',
+  files: [{name: 'file.txt', size: '150'}],
```
Comment (TomokoMiyake) [inline docs/document/website/versioned_docs/version-1.8.0/components/desktop/attachment.md:69]: How about adding File object pattern?

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]

**Review by TomokoMiyake — APPROVED** [review]
I fixed some more points. Please confirm it.

File: docs/document/website/translated_docs/ja/version-1.8.0/components/desktop/attachment.md [hunk 5]
```diff
@@ -7,4 +7,83 @@ original_id: attachment
...
+
+> *1: "参照" ボタンと "ここにファイルをドロップします" のテキストは language プロパティの値に応じて切り替わる.
+
+### Event
+
+指定できるイベントの一覧です。
+
+| Name | Type | Description | Remark |
+| :--- | :--- | :--- | :--- |
+| change | function | ファイルが変更された時のイベントハンドラー | 引数には Event の event オブジェクトをとる<br>event.detail で以下の値を受け取ることができる<li>add-file（ファイル追加された時にトリガーされる）<ul><li>event.detail.type: "add-file"<li>event.detail.oldFiles: 追加前のファイルリスト<li>event.detail.files: 追加後のファイルリスト<li>event.detail.fileIndex: 追加されたファイルの配列番号（Type: Array<number>）<ul><li>"event.detail.files[event.detail.fileIndex[0]]" で追加されたファイルを取得できる</ul></li></ul><li>remove-file（ファイル削除された時にトリガーされる）<ul><li>event.detail.type: "remove-file"<li>event.detail.oldFiles: 削除前のファイルリスト<li>event.detail.files: 削除後のファイルリスト<li>event.detail.fileIndex: 削除されたファイルの配列番号（Type: Array<number>）<ul><li>"event.detail.oldFiles[event.detail.fileIndex[0]]" で削除されたファイルを取得できる</ul></li></ul></li> |
```
Comment (cy-kaneko) [inline docs/document/website/translated_docs/ja/version-1.8.0/components/desktop/attachment.md:46]: 表記ゆれ

> ファイルが変更された時のイベントハンドラー 

ファイルが変更された時のイベントハンドラ

**Pull request merged**